### DISCUSSION TITLE
谢谢

### DISCUSSION THREAD

**Discussion by jicai** [getlantern/lantern-forum-cn#112] [General]
开始的时候，既可以chatgpt，也可以google，一切都是那么美好。
然后就是，不可以chatgpt，可以google，崩溃
然后切回了SW服务器，一切又到好了。
今天一看，我晕，又硬性指定到了US服务器上了，chatgpt跟google都不行了，

你们这是折腾什么啊，直接在服务器显示区域后面放个下拉框，让用户自己选择服务器不就好了，至于是选择美国的服务器、新加坡的服务器、香港的服务器亦或者是大陆内的服务器，让我们自己选择可以吗？别担心服务器撑爆导致用户体验不好了，你们现在的体验已经是非常的差劲了。

亦或者，难道是有团队在背后整你们吗？你们最近惹什么不该惹的人了吗？

**Comment by MaxMKMax** [comment]
@jicai 您好。我有和团队反馈您们需要多个服务器选项了。免得被卡在一个无法使用的服务器里。也有和他们反馈最近不稳定的问题。

团队已经重视你们的意见了，正在开始计划弄个UI给你们选你们要的服务器。要等一小段时间就会launch了。不好意思啊，希望你能够等待。谢谢您。

**Reply by jicai** [reply]
今天早晨更新了下版本，Github、Google都可以访问了，ChatGPT也没有问题，服务器指定的是HK，谢谢！
